pkg/api/defaults_test.go

```diff
@@ -197,6 +197,7 @@ func TestAssignDefaultAddonImages(t *testing.T) {
 		IPMASQAgentAddonName:               "k8s.gcr.io/ip-masq-agent-amd64:v2.0.0",
 		AzureCNINetworkMonitoringAddonName: "containernetworking/networkmonitor:v0.0.4",
 		DefaultDNSAutoscalerAddonName:      "k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.1.1",
+		DefaultHeapsterAddonName:           "k8s.gcr.io/heapster-amd64:v1.5.1",
```

Comment:
This wasn't defined before. Since it was assumed as an always present add-on, it was never configurable by the user externally. 

In the new no-sed implementation, we need these addons to be present so that they can auto-populated by defaults.